index.html

```diff
@@ -285,8 +291,8 @@
     "PerformanceObserver">
       <h2>The <dfn>PerformanceObserver</dfn> interface</h2>
       <p>The <a>PerformanceObserver</a> interface can be used to observe the
-      <a>Performance Timeline</a> and be notified of new performance entries as
-      they are recorded.</p>
+      <a>Performance Timeline</a> to be notified of new performance entries as 
+      they are recorded, and optionally buffered performance metrics.
```

Comment:
I'm struggling with this sentence.. hard to parse. 

"and, optionally, previously buffered performance metrics of specified type."  =/